source/main/utils/buildAppMenus.ts

```diff
@@ -64,7 +63,10 @@ export const buildAppMenus = async (
     logger.info('Exiting Daedalus with code 21', {
       code: 21,
     });
-    safeExitWithCode(21);
+    // We have to make sure that cardano-node exits, otherwise we get DB locked errors at startup:
+    // WRONG: safeExitWithCode(21);
```

Comment:
Looks correct, safeExit is triggered when the Window is closed via `safeExit`, so no need to comment out the old code with `safeExitWithCode`

Reply:
Fixed in ab60ea2c8db169fa2c4e1fa149d6592059b80765